modules/apigee_edge_apidocs/src/Form/ApiDocSettingsForm.php

```diff
@@ -41,13 +42,55 @@ public function getFormId() {
    * {@inheritdoc}
    */
   public function submitForm(array &$form, FormStateInterface $form_state) {
+    $entity_type = \Drupal::entityTypeManager()
```

Comment:
Inject the entity type manager service.

Reply:
Added njecting the entity type manager service in the settings form.